_posts/2021-11-07-canary.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: "Use a canary in your Apache Kafka coal mine"
+date: 2021-11-08
+author: paolo_patierno
+---
+
+The Strimzi cluster operator is really great to deploy your Kafka cluster on Kubernetes and handling everything related to it.
+It helps with brokers configuration, managing the security aspects, doing the needed upgrades, scaling up and down and many more.
+But ... what if you want to have more insights about how the Kafka cluster is behaving?
+What if you want to know if the Kafka brokers are operating correctly so that clients can exchange messages?
+
+This is where a new component in the Strimzi family comes into the picture ... please welcome to the canary!
+
+<!--more-->
+
+## How a canary can help in a coal mine?
+
+When a Kafka cluster is successfully deployed using the Strimzi operator, the next step is to use it with your own applications or maybe, depending on your business, allowing external users to use it even for integration with your solution.
```

Comment:
Maybe _depending on your use-case_ might be better than _depending on your business_?